tools/docker/bootstrap-mercurial.sh

```diff
@@ -12,7 +12,6 @@ python2 setup.py install
 # Setup mercurial with needed extensions
 hg clone -u cc7be0763bb7cb36e64b55b8cec6998741709776 https://hg.mozilla.org/hgcustom/version-control-tools /src/version-control-tools/
 mkdir -p /etc/mercurial/hgrc.d
-ln -s /src/events/docker/hgrc /etc/mercurial/hgrc.d/code-review.rc
```

Comment:
Couldn't we keep this so we use the same hgrc in the integration test and in events?

Reply:
I did that to have dedicated mercurial configuration for both events & integration test (integration only need robustcheckout really)

Reply:
Having more extensions than strictly needed doesn't hurt, and we avoid having unnecessary duplication. It's a nit though, feel free to ignore ;)

Reply:
Done